test/sidekiq-monitor-stats_test.rb

```diff
@@ -30,6 +31,18 @@ def test_with_some_data
       }
 
       conn.hmset('foo:1234', 'info', Sidekiq.dump_json(process_stats), 'at', @started_at.to_f, 'busy', 4)
+
+      job_stats = {
```

Comment:
I referenced this test.
https://github.com/mperham/sidekiq/blob/master/test/test_web.rb#L70L86